docs/source/error_code_list2.rst

```diff
@@ -255,3 +255,31 @@ no error in practice. In such case, it might be prudent to annotate ``items: Seq
 This is similar in concept to ensuring that an expression's type implements an expected interface (e.g. ``Sized``),
 except that attempting to invoke an undefined method (e.g. ``__len__``) results in an error,
 while attempting to evaluate an object in boolean context without a concrete implementation results in a truthy value.
+
+
+Check that ``# type: ignore`` include an error code
```

Comment:
```suggestion
Check that ``# type: ignore`` include an error code [ignore-without-code]
```

Include the error code in the section title, as with the previous section

Reply:
f9e60f9b36eda3443a69481bbffed9e293c87833 adds this